src/Templates/Boilerplate/Bit.Boilerplate/src/Client/Boilerplate.Client.Core/Components/Layout/AppAiChatPanel.razor.scss

```
@@ -58,6 +58,64 @@
+            align-items: flex-end;
+        }
+    }
+
+    // Always laid out and only faded, so revealing it never reflows the conversation - on a pane that auto-scrolls to
+    // the bottom, a reflow would move the very message being pointed at. min-height holds the space while an answer
+    // is still streaming and the buttons are not rendered yet.
+    .message-actions {
+        opacity: 0;
+        min-height: 1.25rem;
+        transition: opacity 0.15s ease-in-out;
+        // Only as wide as its buttons, so the row's own alignment places it: under the start of an answer, and under
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)